### PR TITLE
Changes to lavaland sin ruins

### PR DESCRIPTION
**Pull request by lordpidey**
Adds lavaland wrath ruin.
Adds lavaland acedia ruin
Lavaland ruins can now cause sintouching.
People who've sold their soul can use a pitchfork without burning themselves.

wrath has an altar of wrath. Upon using the altar, you will be equipped with a NODROP hand of wrath, which is a one time use melee weapon that gibs you, AND your target.

Acedia simply has you sell your soul, and get a pitchfork.

TOKEN:  #18143

### CODE REVIEW

**Comment by Kromgar** [comment]
What is sintouching?

File: code/modules/ruins/lavaland_ruin_code.dm [hunk 1]
```diff
@@ -158,8 +181,23 @@
...
+	name = "note from sloth"
+	var/used = 0
+	desc = "have not gotten around to finishing my cursed item yet sorry - sloth"
+
+/obj/item/weapon/paper/sloth/examine(mob/user)
+	..()
+	if(!used && istype(user, /mob/living/carbon/human))
+		used = 1
+		var/mob/living/carbon/human/H = user
+		H.influenceSin(PRIDE)
```
Comment (ChangelingRain) [inline code/modules/ruins/lavaland_ruin_code.dm]: Should be SLOTH

**Comment by lordpidey** [comment]
Sintouched are very minor 'antags' similar to abductees.  They are related to the devil antag.

File: code/modules/mob/living/carbon/human/human.dm [hunk 2]
```diff
@@ -1033,28 +1033,28 @@
 			dna.remove_mutation(HM.name)
 	..()
 
-/mob/living/carbon/human/proc/influenceSin()
+/mob/living/carbon/human/proc/influenceSin(sintype = rand(1,7))
```
Comment (KazeEspada) [inline code/modules/mob/living/carbon/human/human.dm]: Another human proc!

Comment (lordpidey) [inline code/modules/mob/living/carbon/human/human.dm]: Should this be moved to carbon?

**Comment by Bawhoppen** [comment]
>unironically using acedia meme

no

**Comment by Core0verload** [comment]
I don't like new sintouching stuff.

**Comment by InsaneHyena** [comment]
> acedia
End this meme, please.

**Pull request closed without merging**